vllm/model_executor/layers/fused_moe/configs/E=8,N=1792,device_name=NVIDIA_A100-SXM4-80GB.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
nit: Could you use the previous format where we put the config dict for each batch size in a single row? I think that's better for readability.

Reply:
imo this would be better though. They are generated by code `json.dump(existing_content, f, indent=4)`. Generating one for each line is not easy to set up with Python.

Reply:
I see. Makes sense.